app/assets/javascripts/application.js

```diff
@@ -22,6 +22,14 @@
 
   var module;
 
+  if(typeof console === 'undefined') {
```

Comment:
That depends on whether the module loader should go in because it's the only JavaScript that uses the `console` object. Thoughts?